multimodal/src/autogluon/multimodal/optimization/lit_distiller.py

```diff
@@ -52,6 +55,9 @@ def __init__(
         warmup_steps: Optional[int] = None,
         hard_label_loss_func: Optional[_Loss] = None,
         soft_label_loss_func: Optional[_Loss] = None,
+        output_feature_adaptor: Optional[nn.Module] = None,
+        output_feature_loss_func: Optional[_Loss] = None,
+        rkd_loss_func: Optional[nn.Module] = None,
```

Comment:
Add these to the `ignore` list at the beginning of init.

Reply:
Done